src/bundle/Resources/config/ezplatform_default_settings.yml

```diff
@@ -27,3 +27,6 @@ parameters:
 
     # User identifier
     ezsettings.default.user_content_type_identifier: ['user']
+
+    # Copy subtree children limit
+    ezsettings.default.application_config.copy_subtree.limit: 100
```

Comment:
@SylvainGuittard @bdunogier 
In a wider context, related to [EZP-29026](https://jira.ez.no/browse/EZP-29026) WDYT about using common setting for all subtree operations, e.g. `subtree_operations.limit`. The performance issues we encounter are rather the same in case both copying subtree and assigning section and other possible changes that influence search index and caching.

Reply:
I think it's a valid point.
+1

Reply:
I had a discussion with @andrerom yesterday and he said that assigning section is not a "heavy" operation. So we can maybe keep this setting separated.

Reply:
I think this should be separated for better flexibility.